tests/framework/installer/ceph_manifests_v1.2.go

```diff
@@ -1872,7 +1873,7 @@ spec:
     type: s3
     sslCertificateRef:
     port: ` + strconv.Itoa(port) + `
-    securePort:
+    securePort: ` + strconv.Itoa(securePort) + `
```

Comment:
Until we add a test that uses the securePort, I'd suggest just deleting this line from the test and not passing in the securePort argument.
```suggestion
```

Reply:
Done.